### PR TITLE
Register Synternet IBC information

### PR DESCRIPTION
**Pull request by foxis**
Registering Testnet and Mainnet IBC channels.

### CODE REVIEW

File: testnets/_IBC/synternettestnet-osmosistestnet.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{
+  "$schema": "../ibc_data.schema.json",
```
Comment (JeremyParish69) [inline testnets/_IBC/synternettestnet-osmosistestnet.json:2]: this relative path requires and an extra ../ (to go up two directories)
`"$schema": "../../ibc_data.schema.json",`

Comment (foxis) [reply testnets/_IBC/synternettestnet-osmosistestnet.json]: Thanks! Fixed.

File: _IBC/synternet-osmosis.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "../ibc_data.schema.json",
+  "chain_1": {
+    "chain_name": "synternet",
```
Comment (JeremyParish69) [inline _IBC/synternet-osmosis.json:4]: I overlooked this before, but the chains have to be in alphabetical order (for chain_1 vs chain_2 and the file names)

Comment (foxis) [reply _IBC/synternet-osmosis.json]: Updated.

**Pull request merged**